src/renderer/Util.ts

```diff
@@ -122,3 +122,16 @@ export function getFileExtension(filename: string): string {
   if (firstDot === -1) { return ''; }
   return filename.substr(firstDot);
 }
+
+/**
+ * Create a new array and populate it with the properties and values from another array or array like object.
+ * @param arrayLike Array or array like object to copy properties and values from.
+ * @returns New array with the same properties and values as the argument.
+ */
+export function copyArrayLike<T>(arrayLike: { [key: number]: T }): Array<T> {
```

Comment:
Is there any reason to keep this function? It is used twice to copy a ``FileList``, but the copies are not used for anything that requires it to be copied (as far as I cal tell).

Reply:
I was working with the drop functions that already existed, so I moved it to util so I could use it in CurateBox, assuming there was a good reason it was needed to begin with. Can strip it if it's not.